resources/mac/README.md

```diff
@@ -0,0 +1,205 @@
+````
+$ git clone https://github.com/ec-europa/joinup-dev.git
+````
+
+2 - Create file **build.properties.local** in the project with content
+```
+# The location of the Composer binary.
+composer.bin = /usr/local/bin/composer
+
+# Database settings.
+drupal.db.name = joinup
+drupal.db.user = root
+drupal.db.password = 
+
+# Admin user.
+drupal.admin.username = admin
+drupal.admin.password = admin
+
+# The base URL to use in Behat tests.
+behat.base_url = http://joinup.test/
+drupal.base_url = http://joinup.test/
+
+# Paths
+isql.bin = /usr/local/bin/isql
+sparql.dsn = localhost
+sparql.user = dba
+sparql.password = dba
+
+# Migration.
+migration.db.name = joinup6
+migration.db.user = joinup6
+migration.db.password = joinup6
+
+# Piwik configuration
+piwik.db.password = password
+piwik.port = 80
+piwik.website_id = 1
+piwik.url.http = http://piwik.test/
+
+# The credentials of the S3 bucket containing the databases.
+# Production db dumps.
+#exports.sql.source = joinupv2.0/dumps/prod/joinup-full-20180220.sql
+
+# Virtuoso
+virtuoso.binary = /usr/local/bin/virtuoso-t
+
+# Solr
+solr.download.url = https://archive.apache.org/dist/lucene/solr/7.7.1/solr-7.7.1.tgz
+
+# ASDA settings
+asda.username = ''
+asda.password = ''
+exports.virtuoso.source = ''
+exports.sql.source = ''
+exports.solr.filename = ''
+exports.solr.source = ''
+exports.s3.bucket = ''
+exports.s3.key = ''
+exports.s3.secret = ''
+```
+**! Important: For the ASDA settings please contact your local developer !**  
+
+2 - Run composer
+````
+$ composer install
+````
+
+3 - Run build dev 
+```
+$ ./vendor/bin/phing build-dev
+```
+
+4 - install and/or relink
+```
+$ brew unlink unixodbc
+$ brew install virtuoso
+$ brew unlink virtuoso
+$ brew link unixodbc
+$ brew link --overwrite virtuoso
+```
+
+5 - setup viriuoso
+```
+$ ./vendor/bin/phing virtuoso-setup
+$ ./vendor/bin/phing virtuoso-start
+$ ./vendor/bin/phing setup-virtuoso-permissions
+```
+
+6 - Run install dev
+```  
+$ ./vendor/bin/phing install-dev
+```
+
+7 - Setup solar and check if its running
+```  
+$ ./vendor/bin/phing setup-apache-solr
+```
+[Check Virtuoso](http://localhost:8890/sparql)  
+[Check Solr](http://localhost:8983/solr/#/)
+
+8 - Download Databases
+````
+ $ ./vendor/bin/phing download-databases
+````
+
+9 - Rebuild environment
+```  
+$ ./vendor/bin/phing install-dev
+```
+
+10 - Enable developers settings
+```  
+ $ ./vendor/bin/phing setup-dev   
+```
+**WARNING** If you get an error:  
+*“ Your site configuration active store is currently locked.”*  
+comment this line in **/web/sites/default/settings.php**  
+$settings['config_readonly'] = !file_exists(getcwd() . '/../disable-config-readonly');
+
+11 - unblock the admin user
+````
+drush user:unblock
+````
+
+12 - create a new admin user 
+````
+drush uli
+````
+
+## Using the gulp
+For using the gulp please set your node version to v11.15.0 ( its recommanded to know your current node version so your can put ik back for newer projects ).
```

Comment:
For the moment, I will remove the Gulp part which should be part of the existing [FRONT-END documentation](https://github.com/ec-europa/joinup-dev/blob/develop/web/themes/joinup/README.md). 
An update should therefore be made on this part of the documentation.

There is also an open PR for updating Gulp and these dependencies: https://github.com/ec-europa/joinup-dev/pull/2005

Reply:
Oke